CMSgov/design-system
# [WNMGDS-2139] Sass token distribution & Build Updates

## Summary

[WNMGDS-2139](https://jira.cms.gov/browse/WNMGDS-2139) 

- Updates to token system to build Sass token variables and pass them on to `dist` folders during build
- A couple small naming changes in token scripts
- Removes datestamp for token-system-generated files, too much noise updating these every time. I will have another bit of work to re-add this functionality in the future as a custom git commit hook instead so the datestamp is applied only when those files are different than the current checked in versions.

## How to test

1. `yarn clean && yarn install && yarn build`
2. Check /dist/scss folders for token scss
3. Any other standard tests

## Checklist
